### DISCUSSION TITLE
Селект по созданным данным в другом поле этой же таблицы

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#571] [Как это сделать?] [answered]
Подскажите если возможно такое:

есть некая временная таблица.
в ней есть поле в хедере: h_select

есть поле в строчной части: select_field

можно ли в поле h_select кодом создать данные для selectRowListForSelect
а поле select_field сделать с типом Селект и,не создавать вручную селект, а вытащить данные из поля в хедере?

**Comment by Re-Requeste** [comment, marked as answer]
Что первое пришло в голову : 
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/35e252c7-7ea6-4b5a-bdf6-c457e5186316)

Код в h_select: 
```
= :   rowlistCreate(field: 'value' = $value; field: 'title' = $title; field: 'is_del' = false)
	value: listCreate(item: "1"; item: "2"; item: "3")
    title: listCreate(item: "Alexey"; item: "Pavel"; item: "Margo")
```

Код в select_field
`
#h_select
`

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/cdc4cb7c-73ff-4a74-abfe-05a2bba5a480)

Получаем это : 
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/9b154566-8b3e-49a1-8902-a716ddc75137)


Использованная документация: 


`(https://ru.docs.totum.online/relational-associations)`
`(https://ru.docs.totum.online/functions#fn-rowListCreate)`
`(https://ru.docs.totum.online/functions#fn-selectRowListForSelect)`

**Reply by mihail-cornev** [reply]
Спасибо за помощь. Получилось сделать динамические названия полей в канбан, благодаря вашей помощи. 

![Screenshot_14](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/65efdfad-cc12-450b-88d8-03b6d05677eb)
